src/sentry/static/sentry/app/views/projectReleases/releaseLanding.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {t} from 'app/locale';
+import {
+  BashCard,
+  Issues,
+  SuggestedAssignees,
+  Emails,
+  Contributors,
+} from 'sentry-dreamy-components';
+import ReleaseLandingCard from 'app/views/projectReleases/releaseLandingCard';
+import withApi from 'app/utils/withApi.jsx';
+
+const cards = [
+  {
+    title: t("You Haven't Set Up Releases!"),
+    message: t(
+      'Releases provide additional context, with rich commits, so you know which errors were addressed and which were introduced for the first time'
+    ),
+    component: Contributors,
+  },
+  {
+    title: t('Suspect Commits'),
+    message: t(
+      'Sentry suggests which commit caused an issue and who is likely responsible so you can triage'
+    ),
+    component: SuggestedAssignees,
```

Comment:
to fix the bug you are encountering, try styling this and passing in the styled component. something like:

```js

const StyledSuggestedAssignees = styled(SuggestedAssignees)`
  width: 200px;
  height: 200px;
`;

// and in the cards array:

component: StyledSuggestedAssignees;

```

The illustrations are designed to be flexible and fill their parent container. That way they work on mobile devices etc. I think what is happening is that the `SuggestedAssignees` component can't be inside a container with a non-square width because it will screw up the relative transforms.

Reply:
You might also be able to use this trick to limit the space between the illustration and the text:

![image](https://user-images.githubusercontent.com/435981/46507615-3f1fe980-c7ee-11e8-8497-cfe98b23f760.png)

Reply:
I actually take that last one back. I think that is just `align-items: center` at work.